<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatchScreen</class>
 <widget class="QWidget" name="MatchScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>771</width>
     <height>551</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="CourtScreen">
    <widget class="QGraphicsView" name="court">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>700</width>
       <height>400</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
    <widget class="QLabel" name="commentary">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>490</y>
       <width>691</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="scoreOne">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>70</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="scoreTwo">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>10</y>
       <width>70</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="clockMinutes">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="digitCount">
      <number>2</number>
     </property>
    </widget>
    <widget class="QLCDNumber" name="clockSeconds">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="digitCount">
      <number>2</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Filled</enum>
     </property>
     <property name="value" stdset="0">
      <double>0.000000000000000</double>
     </property>
     <property name="intValue" stdset="0">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLCDNumber" name="clockShot">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="digitCount">
      <number>2</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="TacticScreen">
    <widget class="QComboBox" name="positionOne">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="strategyOne">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>60</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>661</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="strategyTwo">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>100</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="positionTwo">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="strategyThree">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>140</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="positionThree">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="strategyFour">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>180</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="positionFour">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>180</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="strategyFive">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>220</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="positionFive">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>220</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="swapButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>swap</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="playerOne">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="playerTwo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="playerThree">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="playerFour">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="playerFive">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="matchButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Match</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tacticsButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Tactics</string>
   </property>
  </widget>
  <widget class="QSlider" name="matchSpeed">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="tracking">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>1</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
